### PR TITLE
Add release notes, pyGPCCA mentions

### LINKED ISSUE #476: 1.2 Release
to do's before this release
- [ ] finish pyGPCCA integration, make sure tests pass (#472)
- [ ] close the legend plotting PR (#470)
- [ ] write release notes 

I will suggest text for the release notes below.

### PR DESCRIPTION
**Pull request by michalk8**
**IMPORTANT: Please search among the [Pull request](../pulls) before creating one.**

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)

## Closes

closes #476 #477 #478

TODO:
~~merge #480 and merge https://github.com/theislab/cellrank/pull/470~~ done

### CODE REVIEW

File: cellrank/tl/estimators/_gpcca.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 @d.dedent
 class GPCCA(BaseEstimator, Macrostates, Schur, Eigen):
     """
-    Generalized Perron Cluster Cluster Analysis [GPCCA18]_.
+    Generalized Perron Cluster Cluster Analysis [GPCCA18]_ and implemented in `pyGPCCA <https://pygpcca.readthedocs.io/en/latest/>`_.
```
Comment (msmdev) [inline cellrank/tl/estimators/_gpcca.py:35]: @michalk8 mb. a typo: "...as implemented..." instead of "...and implemented..."?

**Comment by msmdev** [comment]
@michalk8 @Marius1311 wonderful, thank you very much for the publicity/mentions for pyGPCCA. :)

**Comment by msmdev** [comment]
@michalk8 just one final favor:
could you please rephrase the sentence 

"The Markov chain is coarse-grained into a set of macrostates which represent initial & terminal states as well as transient intermediate states. For each transient cell, i.e. for each cell that's not assigned to a terminal state, we then compute its fate probability of it reaching any of the terminal states."

in the README.rst to

"The Markov chain is coarse-grained into a set of
macrostates which represent initial and terminal states, as well as transient intermediate states using Generalized
Perron Cluster Cluster Analysis (G-PCCA) [GPCCA18]_ as implemented in the novel `pyGPCCA`_ package.
For each transient cell, i.e. for each cell that's not assigned to a terminal state, we then compute its fate
probability of it reaching any of the terminal states."

like in the index.rst as well?

Thank you!

File: docs/source/latest_additions.rst [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
-  benchmarking results in our `CellRank preprint <https://doi.org/10.1101/2020.10.19.345983>`_. This is a reprogramming
-  dataset from [Morris18]_.
+- Completely **refactored the underlying code base of GPCCA** and set it up as it's own package called
+  `pyGPCCA <https://pygpcca.readthedocs.io/en/latest/>`_ with documentation and an example. Going forwards, this will
+  ensure that one of the "engines" of CellRank is also easy to maintain to extend. Further, this will make CellRank's
+  installation more convenient by not needing to vendorize additional dependencies
+  `PR 472 <https://github.com/theislab/cellrank/pull/472>`_.
+- Add :func:`cellrank.pl.circular_projection` visualizing computed fate probabilities as done in [Velten17]_,
+  see :ref:`sphx_glr_auto_examples_plotting_plot_circular_embedding.py`.
+  `PR 459 <https://github.com/theislab/cellrank/pull/459>`_.
```
Comment (Marius1311) [inline docs/source/latest_additions.rst:30]: Nice! Please don't forget to move the old release notes over to the release notes section, i.e. here: https://cellrank.readthedocs.io/en/latest/release_notes.html

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -6,9 +6,33 @@ Release Notes
 Version 1.0
 -----------
 
+1.1.0 :small:`2020-11-17`
+~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (Marius1311) [inline docs/source/release_notes.rst:10]: Okay, ignore my comment above...

**Review by Marius1311 — APPROVED** [review]
Great, work! Ready to be merged.

**Comment by michalk8** [comment]
After a new release is done, the conda recipe must be also modified.

**Comment by Marius1311** [comment]
thanks.

**Pull request merged**